lib/blockchain/chaindb.js

```diff
@@ -1538,6 +1538,35 @@ class ChainDB {
       return undefined;
 
     // Update chain state value.
+    this.updateChainState(entry, block, view);
+
+    // Commit new coin state.
+    this.saveView(view);
+
+    // Write undo coins (if there are any).
+    if (!view.undo.isEmpty())
+      await this.blocks.writeUndo(hash, view.undo.commit());
+
+    // Prune height-288 if pruning is enabled.
+    return this.pruneBlock(entry);
+  }
+
+  /**
+   * Update chain state with coins.
+   * @param {ChainEntry} entry
+   * @param {Block} block
+   * @param {CoinView} view
+   */
+
+  updateChainState(entry, block, view) {
+    const hash = this.network.bip30[entry.height];
+
+    // Blocks 91842 and 91880 created duplicate
+    // txids and should not increment or decrement
+    // the statistics.
+    if (hash && hash.equals(entry.hash))
+      return;
+
```

Comment:
Block 91880 has one other transaction besides the coinbase, won't this cause the second TX to be ignored?

91812 (the first block from the first pair of dupes) also has 2 TXs in the bock, but that'd be ok here.

The other two blocks involved are coinbase-only:

**First pair (both cb txids are `d5d27987d2a3dfc724e359870c6644b40e497bdc0589a033220fe15429d88599`):**
https://blockstream.info/block/00000000000af0aed4792b1acee3d966af36cf5def14935db8de83d6f9306f2f
https://blockstream.info/block/00000000000a4d0a398161ffc163c503763b1f4360639393e0e4c8e300e0caec


**Second pair (both cb txids are `e3bf3d07d4b0375638d5f1db5255fe07ba2c4cb067cd81b84ee974b6585fb468`):**
https://blockstream.info/block/00000000000271a2dc26e7667f8419f2e15416dc6955e5a6c6cdf3f2574dd08e
https://blockstream.info/block/00000000000743f190a18c5577a3c2d2a1f610ae9601ac046a38084ccb7cd721

Reply:
Oh, right. For some reason I thought there were only coinbase txs for the blocks. This should only apply to the coinbase.

Reply:
Should be resolved in the last commit.